src/wp-includes/media.php

```diff
@@ -5455,6 +5459,21 @@ function wp_get_loading_attr_default( $context ) {
 		return false;
 	}
 
+	// Conditionally skip lazy-loading on images before the loop.
+	if ( 'wp_get_attachment_image' === $context || 'the_post_thumbnail' === $context ) {
```

Comment:
See my comment here.

Reply:
We can probably fix this the same way as https://github.com/WordPress/wordpress-develop/pull/4300/files#diff-4d28b6652f0dab208c19aa45e1cf09f3c967060d8cc77502bfcd473a3216b612R5464-R5470. Eventually, both of these could use the same code paths, we can update the second PR whenever the first one has been committed.

Reply:
Updated.